roles/dtc/common/templates/ndfc_create_fabric.j2

```diff
@@ -94,6 +94,8 @@
 {# ------------------------------------------------------ #}
 {# Underlay General Parameters                            #}
 {# ------------------------------------------------------ #}
+  ENABLE_SGT: false
```

Comment:
Once https://github.com/netascode/ansible-dc-vxlan/pull/203 is merged we will need to pull in the changes to this PR to handle versioning between templates for properties like `ENABLE_SGT` and `ENABLE_PVLAN`

Reply:
should be fixed now, updated from develop